e2e/atlas/integrations_test.go

```diff
@@ -0,0 +1,306 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// +build e2e atlas,generic
+
+package atlas_test
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"os/exec"
+	"testing"
+
+	"github.com/mongodb/mongocli/e2e"
+	"github.com/stretchr/testify/assert"
+	"go.mongodb.org/atlas/mongodbatlas"
+)
+
+const (
+	integrationsEntity = "integrations"
+	datadogEntity      = "DATADOG"
+	flowdockEntity     = "FLOWDOCK"
+	newRelicEntity     = "NEW_RELIC"
+	opsGenieEntity     = "OPS_GENIE"
+	pagerDutyEntity    = "PAGER_DUTY"
+	victorOpsEntity    = "VICTOR_OPS"
+	webhookEntity      = "WEBHOOK"
+)
+
+func TestIntegrations(t *testing.T) {
```

Comment:
one major thing here, we may get false positives from running this test in parallel given there can only be one of these integrations per project and we're always running with the same project
I think we should reuse your helper to create a new project here and limit the scope of the test and avoid false possitives